appveyor.yml

```diff
@@ -14,7 +14,7 @@ platform:
 #  - ps: $blockRdp = $true; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
 
 environment:
-    BOOST_ROOT: "C:\\Libraries\\boost_1_64_0"
+    BOOST_ROOT: "C:\\Libraries\\boost_1_69_0"
```

Comment:
I think it boost_1_70_0 is better, because we are checking minimum version of boost to be at least 1.70 in cmake file.